.github/workflows/sanitized_ci.yml

```diff
@@ -253,6 +253,6 @@ jobs:
       - name: Self tests (V compiled with -fsanitize=memory)
         run: |
           ./v -cflags -fsanitize=memory -o v cmd/v
-          ./v -cc tcc test-self -msan-compiler
+          ./v -cc tcc -cflags -fsanitize=memory test-self -msan-compiler
```

Comment:
Same, this change will not do anything, except for files, where `tcc` is not used.